API/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Base image
+FROM node:16
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+COPY package*.json ./
+
+# Install app dependencies
+RUN npm install
+
+# Bundle app source
+COPY . .
+
+# Creates a "dist" folder with the production build
+RUN npm run build
+
+# Start the server using the production build
+CMD [ "node", "dist/main.js" ]
```

Comment:
When we've got testing and linting we might want to convert this to a multi-stage build but I don't know how that fits with github actions at this stage.